src/main/java/com/demoproject/cricketapp/controller/MatchController.java

```diff
@@ -0,0 +1,45 @@
+package com.demoproject.cricketapp.controller;
+
+import com.demoproject.cricketapp.beans.BallEvent;
+import com.demoproject.cricketapp.beans.Match;
+import com.demoproject.cricketapp.beans.Scoreboard;
+import com.demoproject.cricketapp.beans.request.MatchRequest;
+import com.demoproject.cricketapp.beans.response.MatchInfoResponse;
+import com.demoproject.cricketapp.facade.MatchFacade;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("match")
```

Comment:
don't we need to put `/` at the end of the Request Mapping? or else it will directly concat with the mapping without mapping

Reply:
It works for Java 20 without specifying /.